data.py

```diff
@@ -0,0 +1,22 @@
+import mysql.connector
+
+# Setting up the Database
+mydb = mysql.connector.connect(
+  host="localhost",
+  user="admin",
+  passwd="admin",
+  database= 'local_database'
+)
+
+# Setting up the cursor for DB
+mycursor = mydb.cursor(prepared=True)
+
+
+def select_user():
+	sql = "SELECT * FROM info WHERE time = (SELECT max(time) FROM local_database.info ) ORDER BY id, time;"
```

Comment:
Here you are assuming all bikepoints data will have the same 'time' field in the API response. What if the time fields for different bike points are different?

Reply:
Since, I've used datetime.now() function to tfill the 'time' field, the 'time' field during the response will have the same value irrespective of the API call.